packages/xod-client/src/debugger/messages.js

```diff
@@ -14,12 +14,19 @@ export const UPDATING_ARDUINO_DEPENDECIES = 'Updating Arduino packages...';
 export const UPDATING_ARDUINO_DEPENDECIES_SUCCESS =
   'All Arduino packages are updated successfully';
 
-export const TABTEST_GENERATING_CODE = 'Generating C++ code...';
-export const TABTEST_BUILDING = 'Building...';
-export const TABTEST_BUILDING_ERROR = `The generated C++ code contains errors. It can be due to a bad node implementation
+export const WASM_NO_RUNTIME_FOUND = 'Unable to get URL of WASM runtime script';
+export const WASM_BUILDING_ERROR = `The generated C++ code contains errors. It can be due to a bad node implementation
 or if your board is not compatible with XOD runtime code. The original compiler error
 message is above. Fix C++ errors to continue. If you believe it is a bug, report the
 problem to XOD developers.`;
+
+export const TABTEST_GENERATING_CODE = 'Generating C++ code...';
+export const TABTEST_BUILDING = 'Building...';
 export const TABTEST_RUNNING = 'Running tabular test...';
 export const TABTEST_ABORTED = 'The test was interrupted by the user';
-export const WASM_NO_RUNTIME_FOUND = 'Unable to get URL of WASM runtime script';
+
+export const SIMULATION_GENERATING_CODE = 'Generating C++ code for simulation…';
+export const SIMULATION_BUILDING = 'Building…';
+export const SIMULATION_RUNNING = 'Running simulation…';
```

Comment:
Inconsistency between `...` and `…` in the messages. Use either of variant everywhere. I suggest sticking to `...` because we’re using a monospaced font.